frontend/src/components/reports/PendingReportsTab.vue

```diff
@@ -5,73 +5,75 @@
         <h2>Report Details</h2>
       </v-col>
       <v-col cols="12" md="7">
-        <FacilityFilter :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
+        <FacilityFilter v-if="!isEmpty(pendingReports)" :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
       </v-col>
     </v-row>
 
-    <v-skeleton-loader :loading="processing" type="table-tbody">
-      <v-data-table :headers="headers" :items="filteredPendingReports" item-key="surveyResponseReferenceNumber" density="compact">
-        <template #[`item.alertType`]="{ item }">
-          <v-icon v-if="item.alertType === 'Due'" color="warning">mdi-alert</v-icon>
-          <v-icon v-else-if="item.alertType === 'Overdue'" color="error">mdi-alert-circle</v-icon>
-          <span class="pl-1">{{ item.alertType }}</span>
+    <v-skeleton-loader :loading="loading" type="table-tbody">
+      <AppAlertBanner v-if="isEmpty(pendingReports)" type="info">
+        <div>You are up to date with your monthly reports.</div>
```

Comment:
There's no need for a div here. The component already generates a div and p as a wrapper.

Reply:
it is updated.